models/interactives.go

```diff
@@ -40,6 +40,15 @@ type InteractiveUpdate struct {
 	Interactive      Interactive `json:"interactive,omitempty"`
 }
 
+// If getlinkedtocollection is true, i.e visit index page through a collection
+//    disregard other metadata fields and pick only collectionid
+// else
+//    usual filter behaviour
+type InteractiveFilter struct {
+	FilterByAssocCollection bool                 `json:"filter_by_assoc_collection,omitempty"`
```

Comment:
i dont think we need the prefix - just `AssocCollection`?

Reply:
not sure on the name too 😄  maybe `LinkingCollection` - the action it will used from?